packages/core/src/lib/core/decorators/entity/column-index.decorator.ts

```
@@ -110,13 +110,13 @@ export function applyTypeOrmIndex(
  */
 export function applyMikroOrmIndex(
 	target: any,
-	propertyKey: string | undefined,
+	propertyKey: string | symbol | undefined,
 	name: string | undefined,
 	properties: string[] | undefined,
 	options: TypeOrmIndexOptions | undefined
 ) {
 	// Converts provided indexing parameters into MikroORM-compatible index options.
-	const mikroOptions = parseToMikroOrmIndexOptions({ name, properties });
+	const mikroOptions = parseToMikroOrmIndexOptions<Object>({ name, properties });
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)